src/variables.scss

```diff
@@ -33,7 +33,7 @@ $hf-line-height-medium: 1.5;
 
 /* Sizes */
 $hf-gutter: 10px;
-$hf-width-container: 1140px;
+$hf-width-container: 1200px;
```

Comment:
I can see the gutter is 10px so shouldn't the container with be 1180?

Reply:
Actually, it should be exactly 1140 (according to mockups), and inside of main elements we are using 30px paddings, so that's why I decided to do it `1200px` (1140 + 30*2), so paddings inside the block and space between block and browser window would be equal:

![image](https://user-images.githubusercontent.com/8372070/32487184-34a435e6-c3a9-11e7-99c2-0396272177b4.png)

Reply:
I understand that but shouldn't the padding be 10 left and 10 right? I'm just guiding myself by the grid gutter.